interview/backend/conftest.py

```diff
@@ -0,0 +1,27 @@
+import pytest
+
+from players.models import Player
+from django.core.management import call_command
+
+
+@pytest.fixture
+def player(db):
+    return Player.objects.create(nick="John Doe")
+
+
+@pytest.fixture
+def player_bot(db):
+    return Player.objects.create(nick="Sophia", is_bot=True)
+
+
+@pytest.fixture
```

Comment:
Also settings should be overriden in this fixture, because if you run tests with with DEBUG set to False in your django settings this fixture will throw a warning, that test data cannot be generated because DEBUG is set to False. It is also a bug that I need to repair in Quiz app, so my fault. For reference see:

https://github.com/AzenoIT/azeno/blob/master/templates/django/backend/conftest.py

Reply:
done